docs/cookbook/platform-sh.rst

```diff
@@ -24,16 +24,16 @@ and go through checkout. Then, when you will have a project ready, give it a nam
 
 * Add the ``.platform.app.yaml`` file at the root of your project repository
 
-This is how this file should look like for Sylius:
+This is how this file should look like for Sylius (tuned version of the default Platform.sh example):
 
 .. code-block:: yaml
 
     # .platform.app.yaml
     name: sylius
 
-    type: "php:7.0"
+    type: "php:7.1"
     build:
-        flavor: symfony
+        flavor: composer
```

Comment:
Did you notice anything when made this change?

Reply:
No, but Platform.sh defines 'symfony' as deprecated and advices 'composer', therefore :)

Reply:
In here they use 'composer' also: https://docs.platform.sh/frameworks/symfony.html. The Symfony docs itself are behind it seems.

Reply:
Ah here is actually the commit I was searching for: https://github.com/platformsh/platformsh-example-symfony/commit/3de359c88fb5316ff653f0142428e658880f1b65#diff-fe14053ff51cb358f8b709415b2a2d17. Looking at the commit message, it seems 'symfony' is an alias of 'composer'. It's probably more safe to go with 'composer'.